crates/key-value/src/lib.rs

```diff
@@ -98,6 +98,8 @@ impl key_value::HostStore for KeyValueDispatch {
         store: Resource<key_value::Store>,
         key: String,
     ) -> Result<Result<Option<Vec<u8>>, Error>> {
+        spin_telemetry::monotonic_counter!(spin.key_value_get = 1, key = key);
```

Comment:
I think we might be emitting them in traces already but I'm not sure its a good idea to be sending KV store keys by default.

Reply:
Danielle opened a somewhat related issue for being more careful with what we're emitting in telemetry. https://github.com/orgs/fermyon/projects/62/views/1?pane=issue&itemId=79180245

If we wanted to get fancy we could only emit these potentially sensitive values when some sort of local debug flag is set. But, for simplicities sake we're probably best of just not emitting it.

Reply:
Yeah if we don't have a specific need for it lets omit until we can come up with a better approach.